components/connect/integration/block_collection/index.styl

```diff
@@ -2,8 +2,10 @@
   position relative
   height 100%
   min-height block_width
+  min-width block_width
   background-color white
   border-radius connect-border-radius
+  z-index 1
 
   &:after
     display block
```

Comment:
I now remember that this was only working because list view is utilizing some classes that were defined in the connect view component, or something